src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1591,6 +1592,58 @@ public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] strin
             return BigNumber.FormatBigInteger(this, format, NumberFormatInfo.GetInstance(provider));
         }
 
+        private string DebuggerDisplay
+        {
+            get
+            {
+                // For very big numbers, ToString can be too long or even timeout for Visual Studio to display
+                // Display a fast estimated value instead
+
+                // Use ToString for small values
+
+                if (_bits is null || _bits.Length <= 2)
```

Comment:
Nit: add parens:
```suggestion
                if ((_bits is null) || (_bits.Length <= 2))
```

Why `2` rather than say `4` since we have `Int128`/`UInt128` and efficient display for those?

Reply:
Just arbitrarily selected. I didn't think about this.